services/vehicle/src/test/java/com/wedriveu/services/vehicle/boundary/PublisherTest.java

```diff
@@ -0,0 +1,113 @@
+package com.wedriveu.services.vehicle.boundary;
+
+import com.wedriveu.services.shared.rabbitmq.client.RabbitMQFactory;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.unit.Async;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.rabbitmq.RabbitMQClient;
+
+import static com.wedriveu.services.shared.utilities.Constants.BODY;
+
+/**
+ * Created by Marco on 09/08/2017.
+ */
+public abstract class PublisherTest {
```

Comment:
This is not really a test, so please give it a better name. Something like `BaseRabbitMqInteractionClient` or something prettirer...